docs/develop/convert-xml-to-json-manifest.md

```diff
@@ -2,24 +2,24 @@
 title: Convert an add-in to use the unified manifest for Microsoft 365
```

Comment:
Did you tag any of the unified manifest articles with "preview" in the TOC? If so, make sure to remove

Reply:
> Did you tag any of the unified manifest articles with "preview" in the TOC? If so, make sure to remove

Good catch. I'll fix those